packages/server-side-graphql-client/README.md

```diff
@@ -0,0 +1,371 @@
+| `returnFields` | `String`   | `id`       | A graphQL fragment of fields to return. Must match the graphQL return type.                                  |
+| `schemaName`   | `String`   | `public`   | Name of your GraphQL API schema. To override the access-control mechanism, provide `internal` as schemaName. |
+| `extraContext` | `Object`   | `{}`       | Additional context option object that gets passed onto `keystone.executeGraphQL` method.                     |
+
+### createItem
+
+Allows you to create single item.
+
+#### Usage
+
+```js
+const { createItem } = require('@keystonejs/server-side-graphql-client');
+
+keystone.createList('User', {
+  fields: {
+    name: { type: Text },
+    email: { type: String },
+  },
+});
+
+const addUser = async userInput => {
+  const user = await createItem({
+    keystone,
+    listName: 'User',
+    item: userInput,
+    returnFields: `name, email`,
+  });
+  console.log(user); // { name: 'keystone user', email: 'keystone@test.com'}
+};
+
+addUser({ name: 'keystone user', email: 'keystone@test.com' });
+```
+
+#### Config
+
+[Shared Config Options](#shared-config-options) apply to this function.
+
+| Properties | Type                            | Default    | Description              |
+| ---------- | ------------------------------- | ---------- | ------------------------ |
+| `item`     | GraphQL `[listName]CreateInput` | (required) | The item to be inserted. |
+
+### createItems
+
+Allows bulk creation of items.
+
+#### Usage
+
+```js
+const { createItems } = require('@keystonejs/server-side-graphql-client');
+
+keystone.createList('User', {
+  fields: {
+    name: { type: Text },
+    email: { type: String },
+  },
+});
+
+const dummyUsers = [
+  {
+    data: { name: 'user1', email: 'user1@test.com' },
+    data: { name: 'user2', email: 'user2@test.com' },
+  },
+];
+
+const addUsers = async () => {
+  const users = await createItems({
+    keystone,
+    listName: 'User',
+    items: dummyUsers,
+    returnFields: `name`,
+  });
+  console.log(users); // [{name: `user2`}, {name: `user2`}]
+};
+addUsers();
+```
+
+#### Config
+
+[Shared Config Options](#shared-config-options) apply to this function.
+
+| Properties | Type                             | Default    | Description                                                                                    |
+| ---------- | -------------------------------- | ---------- | ---------------------------------------------------------------------------------------------- |
+| `items`    | GraphQL `[listName]sCreateInput` | (required) | The array of objects to be inserted.                                                           |
+| `pageSize` | `Number`                         | 500        | The create mutation batch size. This is useful when you have large set of data to be inserted. |
+
+### getItem
+
+[Shared Config Options](#shared-config-options) apply to this function.
```

Comment:
```suggestion
```

Reply:
Why? It accepts the shared config items.

Reply:
It's in the wrong spot of the file.

Reply:
Good point ;)